distribution/src/release/LICENSE

```diff
@@ -519,21 +519,21 @@ For details, see their respective project links.
 * netflix-commons-util (com.netflix.netflix-commons:netflix-commons-util:0.1.1 - https://github.com/Netflix/netflix-commons)
 * netflix-statistics (com.netflix.netflix-commons:netflix-statistics:0.1.1 - https://github.com/Netflix/netflix-commons)
-* Netty/Buffer (io.netty:netty-buffer:4.1.43.Final - http://netty.io/netty-buffer/)
-* Netty/Codec/DNS (io.netty:netty-codec-dns:4.1.43.Final - http://netty.io/netty-codec-dns/)
-* Netty/Codec/HTTP (io.netty:netty-codec-http:4.1.43.Final - http://netty.io/netty-codec-http/)
-* Netty/Codec/HTTP2 (io.netty:netty-codec-http2:4.1.43.Final - http://netty.io/netty-codec-http2/)
-* Netty/Codec/Socks (io.netty:netty-codec-socks:4.1.43.Final - http://netty.io/netty-codec-socks/)
-* Netty/Codec (io.netty:netty-codec:4.1.43.Final - http://netty.io/netty-codec/)
-* Netty/Common (io.netty:netty-common:4.1.43.Final - http://netty.io/netty-common/)
-* Netty/Handler/Proxy (io.netty:netty-handler-proxy:4.1.43.Final - http://netty.io/netty-handler-proxy/)
-* Netty/Handler (io.netty:netty-handler:4.1.43.Final - http://netty.io/netty-handler/)
-* Netty/Resolver/DNS (io.netty:netty-resolver-dns:4.1.43.Final - http://netty.io/netty-resolver-dns/)
-* Netty/Resolver (io.netty:netty-resolver:4.1.43.Final - http://netty.io/netty-resolver/)
-* Netty/TomcatNative [BoringSSL - Static] (io.netty:netty-tcnative-boringssl-static:2.0.26.Final - https://github.com/netty/netty-tcnative/netty-tcnative-boringssl-static/)
+* Netty/Buffer (io.netty:netty-buffer:4.1.45.Final - http://netty.io/netty-buffer/)
+* Netty/Codec/DNS (io.netty:netty-codec-dns:4.1.45.Final - http://netty.io/netty-codec-dns/)
+* Netty/Codec/HTTP (io.netty:netty-codec-http:4.1.45.Final - http://netty.io/netty-codec-http/)
+* Netty/Codec/HTTP2 (io.netty:netty-codec-http2:4.1.45.Final - http://netty.io/netty-codec-http2/)
+* Netty/Codec/Socks (io.netty:netty-codec-socks:4.1.45.Final - http://netty.io/netty-codec-socks/)
+* Netty/Codec (io.netty:netty-codec:4.1.45.Final - http://netty.io/netty-codec/)
+* Netty/Common (io.netty:netty-common:4.1.45.Final - http://netty.io/netty-common/)
+* Netty/Handler/Proxy (io.netty:netty-handler-proxy:4.1.45.Final - http://netty.io/netty-handler-proxy/)
+* Netty/Handler (io.netty:netty-handler:4.1.45.Final - http://netty.io/netty-handler/)
+* Netty/Resolver/DNS (io.netty:netty-resolver-dns:4.1.45.Final - http://netty.io/netty-resolver-dns/)
+* Netty/Resolver (io.netty:netty-resolver:4.1.45.Final - http://netty.io/netty-resolver/)
+* Netty/TomcatNative [BoringSSL - Static] (io.netty:netty-tcnative-boringssl-static:2.0.28.Final - https://github.com/netty/netty-tcnative/netty-tcnative-boringssl-static/)
 * Netty/Transport/Native/Epoll (io.netty:netty-transport-native-epoll:4.1.36.Final - http://netty.io/netty-transport-native-epoll/)
 * Netty/Transport/Native/Unix/Common (io.netty:netty-transport-native-unix-common:4.1.36.Final - http://netty.io/netty-transport-native-unix-common/)
```

Comment:
Please check if these two dependencies is used. If not used, delete it.

Reply:
netty-transport-native-epoll exists in our pom file, but it was excluded by exclusion tag, so we can say we don't use it.
netty-transport-native-unix-common we don't use it.